ios/Podfile

```diff
@@ -31,6 +31,7 @@ end
 # https://github.com/zoontek/react-native-permissions?tab=readme-ov-file#ios
 setup_permissions([
   'PhotoLibrary', # Required for Gallery read permission
+  'Bluetooth'
```

Comment:
Do you think it would be helpful to add a comment with why we are adding this permission?Like the `PhotoLibrary` above